webapp/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,144 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  ButtonVariant,
+  IconVariant,
+  Input,
+  InputVariant,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { ResponseError } from '../../shared/generated';
+import { authApi } from '../../shared/services/ApiService';
+import { LOGIN_EP_URL, REGISTER_URL, USERS_URL } from '../../shared/urls';
+import './LoginPage.css';
+
+type SubmitStatus = {
+  type: 'success' | 'error' | 'pending';
+  message: string;
+};
+
+export default function LoginPage() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [status, setStatus] = useState<SubmitStatus>({
+    type: 'pending',
+    message: '',
+  });
+
+  async function login() {
+    try {
+      await authApi.authControllerLoginLocalUser({
+        loginUserDto: { username, password },
+      });
+      window.location.href = USERS_URL;
+    } catch (error) {
+      if (error instanceof ResponseError) {
+        if (error.response.status === 403 || error.response.status === 401) {
+          setStatus({
+            type: 'error',
+            message: 'Incorrect username or password',
+          });
+        } else {
+          setStatus({ type: 'error', message: `${error.response.statusText}` });
+        }
+      } else if (error instanceof Error) {
+        setStatus({ type: 'error', message: `${error.message}` });
+      }
+    }
+  }
+  const handleOnSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    login();
```

Comment:
> Promise returned from `login()` is ignored 
What about we put a `.catch((e) => console.error(e))` attached to this `login()`?

Reply:
Yes, you are right. Done